adds storage, user and data management pages (#7814)
aws-amplify/docs: src/pages/[platform]/build-a-backend/auth/manage-users/with-amplify-console/index.mdx

import { getCustomStaticPath } from '@/utils/getCustomStaticPath';

export const meta = {
  title: 'Manage users with Amplify console',
  description:

>     'Manage applications Cognito users and groups with Amplify console',

Reviewer: ```suggestion
    'Manage application's Cognito users and groups with the Amplify console',
```

  platforms: [
    'android',
    'angular',
    'flutter',
    'javascript',
    'nextjs',
    'react',
    'react-native',
    'swift',
    'vue'
  ]
};

export function getStaticPaths() {
  return getCustomStaticPath(meta.platforms);
}

export function getStaticProps(context) {
  return {
    props: {
      platform: context.params.platform,
      meta
    }
  };
}

> The **User management** page in the Amplify Console provides a user-friendly interface for managing your application's users. You can create and manage users and groups, edit user attributes, and suspend users. 

Reviewer: ```suggestion
The **User management** page in the Amplify console provides a user-friendly interface for managing your application's users. You can create and manage users and groups, edit user attributes, and suspend users. 
```

> If you have not yet created a **Auth** resource, visit the [Auth setup guide](/[platform]/build-a-backend/auth/set-up-auth/)

Reviewer: ```suggestion
If you have not yet created an **auth** resource, visit the [Auth setup guide](/[platform]/build-a-backend/auth/set-up-auth/)
```

## Access User management

> After you've deployed your **Auth** resource, you can access the manager on Amplify Console.

Reviewer: ```suggestion
After you've deployed your **Auth** resource, you can access the manager in the Amplify console.
```

  description:
    'Manage applications Cognito users and groups with Amplify console',
  platforms: [
    'android',
    'angular',
    'flutter',
    'javascript',
    'nextjs',
    'react',
    'react-native',
    'swift',
    'vue'
  ]
};

export function getStaticPaths() {
  return getCustomStaticPath(meta.platforms);
}

export function getStaticProps(context) {
  return {
    props: {
      platform: context.params.platform,
      meta
    }
  };
}

The **User management** page in the Amplify Console provides a user-friendly interface for managing your application's users. You can create and manage users and groups, edit user attributes, and suspend users. 

If you have not yet created a **Auth** resource, visit the [Auth setup guide](/[platform]/build-a-backend/auth/set-up-auth/)

## Access User management

After you've deployed your **Auth** resource, you can access the manager on Amplify Console.

1. Log in to the [Amplify console](https://console.aws.amazon.com/amplify/home) and choose your app.
2. Select the branch you would like to access.
3. Select **Authentication** from the left navigation bar.
4. Then, select **User management**.

> <video autoPlay={true} muted={true} loop={true} width="100%" playsInline={true}>

Reviewer 2: A wrapper for this video element has been made for docs use `src/components/Video` please use the `<Video>` component here

Author: updated to `<Video/>`
